fastly/fastly.go

```diff
@@ -5,6 +5,17 @@ import (
 	"encoding"
 )
 
+type BatchOperation string
+
+const (
+	Create BatchOperation = "create"
+	Update BatchOperation = "update"
+	Upsert BatchOperation = "upsert"
+	Delete BatchOperation = "delete"
+
+	BatchModifyMaximumItems = 1000
```

Comment:
Can we include a comment explaining this or pointing to the doc that describes this behavior

Reply:
Unfortunately, this limit isn't documented in our docs. At least I couldn't find it 😞

Reply:
Agreed I could not find it either.  I could at least put a comment explaining that.

Reply:
Changed to name to BatchModifyMaximumOperations to reflect its use between dict items and acl entries.   Operations are used over both.  Also added comments to explain it is not currently documented in the API.